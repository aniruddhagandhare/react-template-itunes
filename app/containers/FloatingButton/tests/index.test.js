/**
 *
 * Tests for FloatingButton
 *
 *
 */

import React from 'react';
import { renderProvider } from '@utils/testUtils';
// import { fireEvent } from '@testing-library/dom'
import { FloatingButtonTest as FloatingButton } from '../index';

describe('<FloatingButton /> container tests', () => {
  // let submitSpy

  beforeEach(() => {
    // submitSpy = jest.fn()
  });
  it('should render and match the snapshot', () => {
    const { baseElement } = renderProvider(<FloatingButton />);
    expect(baseElement).toMatchSnapshot();
  });
});
