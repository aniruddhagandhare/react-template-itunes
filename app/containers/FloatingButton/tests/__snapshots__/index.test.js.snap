// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FloatingButton /> container tests should render and match the snapshot 1`] = `
<body>
  <div>
    <div>
      New Button
    </div>
  </div>
</body>
`;
